docs/guides/vpc-peering.md

```diff
@@ -0,0 +1,91 @@
+# Set up VPC peering
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+If your are using a internal operator load balancer (i.e. you set `operator_load_balancer_scheme: internal` in your cluster configuration file before creating your cluster), you can use VPC Peering to enable your Cortex CLI to connect to your cluster operator from another VPC so that you may run `cortex` commands.
```

Comment:
`If your are using a internal operator load balancer` -> `If you are using an internal operator load balancer`